torch/distributed/tensor/parallel/fsdp.py

```diff
@@ -186,7 +186,7 @@ def _chunk_tensor(
             inner_param,
             rank,
             world_size,
-            torch.cuda.device_count(),
+            torch.accelerator.device_count(),
```

Comment:
Can we use ```torch.get_device_module()``` instead of  ```torch.accelerator```  so that out-of-tree device accelerators also benefit.

Reply:
@albanD  could you give any comments for above case?

Reply:
Yeah, maybe `torch.get_device_module()` is broader because it also covers `cpu`.

Reply:
@albanD  Could you give the comments to similar usage in this case.
I assume, we should promote the torch.accelerator APIs as the unified device-agnostic frontend APIs of PyTorch in community. But, looks like the torch.get_device_module() is able to cover more scenarios, which is not in line with above goal.
May I know, which usage should be followed in future cases? Or, we should improve torch.accelerator to cover all usages?